pkg/apis/duck/v1/subscribable_types.go

```diff
@@ -35,6 +35,12 @@ var _ duck.Implementable = (*Subscribable)(nil)
 //
 // At least one of SubscriberURI and ReplyURI must be present
 type SubscriberSpec struct {
+	// Name is used to identify the original subscription object.
+	// +optional
+	Name *string `json:"name,omitempty"`
+	// Namespace is used to identify the namespace of the original subscription object.
+	// +optional
+	Namespace *string `json:"namespace,omitempty"`
```

Comment:
After reviewing this in more details, I think that we can add the namespace when the new cross ns feature is introduced so that we can place the namespace usage behind feature flag and test it extensively as it's a critical security area.

For now, I would use the channel/subscription namespace instead of this custom field